backend/app/assets/javascripts/spree/backend.js

```diff
@@ -11,7 +11,41 @@
 //= require velocity
 //= require spree
 //= require spree/backend/spree-select2
-//= require_tree .
```

Comment:
I don't think you can do it that was because each Gem gets mounted separately. all 5 gems in 1 repository is only for github-- by the time bundler is working with these, bundler sees them as 5 separate gems, so you can't reference "/backend" explicitly.